js/loader.js

```diff
@@ -35,7 +35,7 @@ var Loader = (function() {
 				// This is done after all the moduels so we can
 				// overwrite all the defined styls.
 
-				loadFile("css/custom.css", function() {
+				loadFile(config.customCss, function() {
```

Comment:
? "all system is crashed"? I've tested it and it works beside the notification in the console like any other resource which could not be loaded, e.g. from a module. 

Tested with:
```
config : {
  customCss: "/iDoNotExists.css"
}
```
Can you please provide an example? If we want to check the path for existing we should add that to the "loadFile" method itself. But I don't see any advantages to the current approach to log the error to the console.